pyntc/devices/asa_device.py

```diff
@@ -25,11 +25,16 @@
 )
 
 
+RE_SHOW_FAILOVER_GROUPS = re.compile(r"Group\s+\d+\s+State:\s+(.+?)\s*$", re.M)
+RE_SHOW_FAILOVER_STATE = re.compile(r"(?:Primary|Secondary)\s+-\s+(.+?)\s*$", re.M)
+
+
 @fix_docs
 class ASADevice(BaseDevice):
     """Cisco ASA Device Implementation."""
 
     vendor = "cisco"
+    active_redundancy_states = {None, "active"}
```

Comment:
Why not make this data structure a list?

Reply:
I like the usage of the set.

Reply:
@mundruid The purpose of the data structure is to do lookups (is x in y?), which performs better in sets than lists